src/controllers/Controller.php

```diff
@@ -13,13 +13,16 @@
 	$config = await Configuration::gen('language');
     $language = $config->getValue();
 	$document_root = must_have_string(Utils::getSERVER(), 'DOCUMENT_ROOT');
-	$localize_style="";
+    $localize_style = '';
 	if (! preg_match('/^\w{2}$/', $language)) {
 		$language = "en";
 	}
 	if (file_exists($document_root."/static/css/locals/".$language."/style.css")){
-		$localize_style = <link rel="stylesheet" href="static/css/locals/".$language."/style.css" />
+		$localize_style = '"static/css/locals/'.$language.'/style.css"';
```

Comment:
I think there is a better fix than this, for example:
`<link rel="stylesheet" href={"static/css/locals/".$language."/style.css"} />`